test/parallel/test-debug-brk-no-arg.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+
+const child = spawn(process.execPath, ['--debug-brk=' + common.PORT, '-i']);
+child.stderr.once('data', common.mustCall(function() {
+  child.stdin.end('.exit');
+}));
+
+child.on('exit', common.mustCall(function(c) {
```

Comment:
In this particular case, I don't think it matters either way. But I'm happy to change it to `close` if that is more idiomatic or avoids problems that I'm currently unaware of in the existing code.